channeldb/waitingproof.go

```diff
@@ -242,7 +243,7 @@ func (p *WaitingProof) Decode(r io.Reader) error {
 	}
 
 	msg := &lnwire.AnnounceSignatures{}
-	if err := msg.Decode(r, 0); err != nil {
+	if err := msg.Decode(r, lnwire.ProtocolVersionTLV); err != nil {
```

Comment:
does this work since this message never had a TLV blob?

Reply:
IIRC, `ioutil.ReadAll` will return `nil` if nothing was read.